src/components/footer/footer.scss

```diff
@@ -1,43 +1,67 @@
 @import '../bootstrap/bootstrap';
 
+$social-icon-size: 2.5rem;
+
 .footer {
   background: $darkgrey;
   color: $white;
 }
 
-.logo path {
+.footer .logo path {
   fill: $gold;
 }
 
-.logo svg {
-  margin-left: 2rem;
+.footer .logo svg {
   padding: 1rem 0;
   height: 7rem;
 }
 
 .footer-top {
+  padding-left: 2rem;
```

Comment:
just a minor nitpick but I think we could do consolidate these
`padding: 0 2rem`

Reply:
I set the left and right padding explicitly because I wasn't sure if setting the top and bottom padding would alter something elsewhere in the codebase. It looks like by default the top and bottom padding is 0, so I will make this change as per your request